packages/alfa-refinement/test/refinement.spec.ts

```diff
@@ -0,0 +1,75 @@
+// import { test } from "@siteimprove/alfa-test";
```

Comment:
Tests should be active.

Reply:
Good catch, completely forgot about those! 🤦